### ISSUE TITLE
Monthly Chat Agenda September (2020-09-07)

### ISSUE THREAD

**Issue by franbuehler** [coreruleset/coreruleset#1869]
This is the Agenda for the Monthly CRS Chats.

The general chat is going to happen on https://owasp.slack.com in the channel #coreruleset on Monday, September 7th, at 20:30 CET. That's the 1st Monday of the month. A separate issue chat is happening at the same location, same time on Monday, September 21st. That's the 3rd Monday of the month.

Items on the Agenda:  (see previous meetings decisions: [here](https://github.com/coreruleset/coreruleset/issues/1853))

## PRs

* #1872 - Eliminate some FPs in 941120 by adding char limit (solves part of #1828)
* #1870 - Fix FP in Postgres SQL Information Leakage Rule 951240 (@franbuehler wants to solve issue #1468)
* #1868 - Migrate PR: RE2 compatibility for 921120 (@franbuehler migrated @allanrbo PR as discussed in [monthly chat meeting July](https://github.com/coreruleset/coreruleset/issues/1836#issuecomment-668031921))
* #1848 Permit "version" string in content-type header
* #1846 FEAT(FTW): use master to get py3 support
* #1845 Stop decoding things twice
* #1843 Added new nextcloud excludes for the "Security & setup warnings" test and session tokens
* #1847 Suppress rule 200002 when Nextcloud mobile app accesses shares through API
* #1817 Fix false positives with rule 942360
* #1794 Fix bypass 942190
* #1793 SQLi using set var at PL1

## Other items

* Draft proposal for side project [CRS Kindergarden](https://gist.githubusercontent.com/dune73/cfa815fc4f1485f3e153749261d69dd7/raw/802e7f5038f077a427923f718367721708f73c3c/crs-kindergarden.txt) - @dune73 accepts feedback until September 10.
* We have very inconsistent usage of the `log` directive in the rules. Should we remove it everywhere since it's supposed to be set in `SecDefaultAction` anyways.
* DoS protection in CRS is depending on the `IP` collection. The primary key of that collection is defined in 901xxx as `IP` + `_` + `User-Agent`. We never considered out DoS protection as really good, but maybe steering around these rules is easier still.
* The publication of the ModSec3 weakness is scheduled for September 13. We have not heard from TW in a while but it sounds as if they were not releasing v3.0.5 with the fix in time. See the [June Meeting](https://github.com/coreruleset/coreruleset/issues/1836#issuecomment-668031921) with some additional infos.
* @theMiddle proposes a tag per rule file in #1866. Let's discuss this.

Feel free to add items as you see fit either above, or below as comments.

## Open Issues - Separate Issues Meeting (Monday, September 21st)

In January 2020, we decided to look into 10 issues at the chat every month. But only after the Other items. Pick the issues before the meeting and list them.

* Issue slot 1: #1785 - XML FPs with 932140
* Issue slot 2: #1884 - Wordpress FP on 932200 when writing articles False Positive 
* Issue slot 3: #1883 - FP on 921110
* Issue slot 4: #1880 - FP in Wordpress
* Issue slot 5: #1875 - FP in Wordpress (951240)
* Issue slot 6: #1874 - FP with PGP in message
* Issue slot 7: #1873 - FP with phpBB
* Issue slot 8: #1867 - FP with 941120 with base64 string
* Issue slot 9: #1864 - FP with 930120
* Issue slot 10: #1864 - "Want to check why the CRS need to block patch and put in ENFORCEMENT way"

## How to get to our slack and join the meeting?

If you are not yet on the OWASP Slack, here is your invite: https://owasp-slack.herokuapp.com/ .

Everybody is welcome to join our community chat.

**Comment by franbuehler** [comment]
**Decisions**

* #1872 - Eliminate some FPs in 941120 by adding char limit (solves part of #1828) (merged)
* #1870 - Fix FP in Postgres SQL Information Leakage Rule 951240 (@franbuehler wants to solve issue #1468) (merged)

**Comment by fgsch** [comment]
> We have very inconsistent usage of the log directive in the rules. - The decision is we drop log from the rules - @fgsch wants to open a PR

Opened https://github.com/coreruleset/coreruleset/pull/1876

**Comment by fgsch** [comment]
> #1846 FEAT(FTW): use master to get py3 support - skipped

Asked @fzipi to rebase the PR to run the Github actions again and check what the status is after https://github.com/coreruleset/ftw/pull/49.
